ci/setup-build-test-gdk.ps1

```diff
@@ -65,7 +66,7 @@ class TestSuite {
 [string] $test_map_path = "/Game/Intermediate/Maps/"
 
 [TestProjectTarget] $gdk_test_project = [TestProjectTarget]::new("git@github.com:spatialos/UnrealGDKTestGyms.git", $gdk_branch, "Game\GDKTestGyms.uproject", "GDKTestGyms", $gyms_version_path, "env:TEST_REPO_BRANCH")
-[TestProjectTarget] $native_test_project = [TestProjectTarget]::new("git@github.com:improbable/UnrealGDKEngineNetTest.git", $gdk_branch, "Game\EngineNetTest.uproject", "NativeNetworkTestProject", $gyms_version_path, "env:NATIVE_TEST_REPO_BRANCH")
+[TestProjectTarget] $native_test_project = [TestProjectTarget]::new("git@github.com:improbable/UnrealGDKEngineNetTest.git", $gdk_branch, "Game\EngineNetTest.uproject", "NativeNetworkTestProject", "", "env:NATIVE_TEST_REPO_BRANCH")
```

Comment:
We shouldnt be using the test gym file to define an override for UnrealGDKEngineNetTest. We could consider adding yet another text file for EngineNetTest but I am not sure if its worth it at this point.